HotelApp/Models/LoginUserDTO.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Security.Principal;
+using System.Threading.Tasks;
+
+namespace HotelApp.API.Models
+{
+    public class LoginUserDTO
+    {
+        [Required(ErrorMessage = "UserName is required")]
```

Comment:
Ove validacije napravi pomoću FluentApi Validations. https://fluentvalidation.net/

Realno na ovak jednostavim stvarima tipa LoginUserDto-a nema potrebe, ali tek tolko da prođeš kroz to. Sa fluent api validacijama si puno fleksibilniji i puno je močnije neko anotacije